package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "eleventy-netlify-boilerplate",
+  "version": "1.0.2",
+  "description": "A boilerplate for building a fast static website with the Eleventy static site generator, for deployment to Netlify.",
+  "scripts": {
+    "build": "npx eleventy",
+    "watch": "npx eleventy --watch",
+    "serve": "npx eleventy --serve",
+    "debug": "DEBUG=* npx eleventy",
+    "sass": "node-sass --watch _includes/assets/sass/styles.scss _includes/assets/css/styles.css"
```

Comment:
I would like it if we could build/serve/watch the entire site with one command for easier deployment and better dev experience. I think the way we did this last time was to use a eleventy plugin for postcss, it looks like there's one for Sass too https://www.11ty.dev/docs/plugins/#unofficial-plugins

Reply:
@lewisnyman Great point! I installed this, however I could not seem to get the config working. The plugin is dependant on 11ty and states that if that changes, then this could break. I wonder if that's happened. In the meantime, I have tweaked the serve script so that we serve and build the css concurrently.